### DISCUSSION TITLE
[Feature] Cleanup old Photos that have been synced to Immich

### DISCUSSION THREAD

**Discussion by TiloHeidasch** [immich-app/immich#7621] [Feature Request]
### The feature

I am using the Immich Mobile Apps on both Android and iOS.
I am saving all my Photos to immich with either device.
My storage on my devices keeps running low, so I would like for the immich mobile app to free up space for me.
I would love an option within the mobile app that enables the app to manage my data.
I want the immich app, to delete any local picture (if the option is enabled), that has been saved to immich and is older than 6 Months.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #165
